### PR TITLE
person updates

### PR DESCRIPTION
**Pull request by nickpell**
@proteusvacuum @czue

### CODE REVIEW

File: custom/enikshay/nikshay_datamigration/factory.py [hunk 1]
```diff
@@ -109,11 +108,9 @@ def get_person_case_structure(self):
                     'dob_known': 'no',
                     'first_name': self.patient_detail.first_name,
                     'last_name': self.patient_detail.last_name,
-                    'middle_name': self.patient_detail.middle_name,
                     'name': self.patient_detail.pname,
                     'nikshay_id': self.nikshay_id,
-                    'permanent_address_district_choice': self.patient_detail.Dtocode,
-                    'permanent_address_state_choice': self.patient_detail.scode,
+                    'person_id': 'FROM_NIKSHAY_' + self.nikshay_id,
```
Comment (czue) [inline custom/enikshay/nikshay_datamigration/factory.py]: do we need to get any requirements input on this?

**Review by czue — APPROVED** [review]
:+1:

File: custom/enikshay/nikshay_datamigration/factory.py (hunk 1 above)
Comment (proteusvacuum) [inline custom/enikshay/nikshay_datamigration/factory.py]: [12:27:12 PM] Kriti Mehrotra: hmm
[12:28:41 PM] Farid Rener: or person.person_id = "(from Nikshay) {nikshay_id}"
[12:28:52 PM] Farid Rener: like we have for "test ids"
[12:29:41 PM] Kriti Mehrotra: yeah, just some sort of prefix will help I think
[12:29:47 PM] Farid Rener: cool beans

Comment (czue) [inline custom/enikshay/nikshay_datamigration/factory.py]: yeah sorry i saw that thread. not sure kriti counts and this seems a little ridiculous to show to users

Comment (czue) [inline custom/enikshay/nikshay_datamigration/factory.py]: (but maybe we don't show this to users? i can't keep track anymore...)

Comment (proteusvacuum) [inline custom/enikshay/nikshay_datamigration/factory.py]: I see. Will verify.

Comment (proteusvacuum) [inline custom/enikshay/nikshay_datamigration/factory.py]: 👍 I think we can leave it like this according to reqs.

Comment (czue) [inline custom/enikshay/nikshay_datamigration/factory.py]: thanks for checking!

**Pull request merged**